### DISCUSSION TITLE
Escape keywords

### DISCUSSION THREAD

**Discussion by magnus-madsen** [flix/flix#7862] [Ideas]
We need a mechanism to escape keywords.

For example, we may want to write:

```scala
let v = 150ii;
v.and(-100ii) == 148ii
```

but `and` is a Flix keyword. A solution could be to treat `%and%` as a Java name.

**Comment by JonathanStarup** [comment]
So this is specifically for interop?

**Reply by magnus-madsen** [reply]
Unfortunately, we need something. Otherwise there will be methods you cannot call :[

**Reply by JonathanStarup** [reply]
Hmm you could also just require imports for those few - surely its not that common?

**Reply by magnus-madsen** [reply]
We plan to remove the import feature.

**Reply by JonathanStarup** [reply]
`java alias jand => and;` perhaps, but now its not less work yea
